File: challenges/2-idade-em-dias/typescript/joao-erik2077/idadeEmDias.ts
const diasDeVida = Number(process.argv.slice(2, 3));

const anos = Math.floor(diasDeVida / 365);
const meses = (Math.floor(diasDeVida / 30)) % 12;
const dias = (diasDeVida % 365) % 30;

console.log(`
${anos} ano(os)
${meses} mes(es)
${dias} dia(s)
`);

File: challenges/2-idade-em-dias/python/Victordiash/idade_em_dias.py
idade_inicial = int(input("coloque sua idade em dias, para visualizar ela em anos, meses e dias :"))
idade_ano = idade_inicial // 365
idade_meses = (idade_inicial % 365) // 30
idade_dias = (idade_inicial % 365) % 30

print(f" você possui {idade_ano} ano(s) de idade")
print(f" você possui {idade_meses} mes(es) de idade")
print(f" você possui {idade_dias} dia(s) de idade")